diff --git a/node_modules/three-spritetext/dist/three-spritetext.common.js b/node_modules/three-spritetext/dist/three-spritetext.common.js
index 5c2ecf9..658c4db 100644
--- a/node_modules/three-spritetext/dist/three-spritetext.common.js
+++ b/node_modules/three-spritetext/dist/three-spritetext.common.js
@@ -136,8 +136,7 @@ function _nonIterableSpread() {
   throw new TypeError("Invalid attempt to spread non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.");
 }
 
-var three = window.THREE ? window.THREE // Prefer consumption from global THREE, if exists
-: {
+var three = {
   LinearFilter: three$1.LinearFilter,
   Sprite: three$1.Sprite,
   SpriteMaterial: three$1.SpriteMaterial,
